index.html

```diff
@@ -10,23 +10,234 @@
+        <ul class="languages-list">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <div class="project-access">See project</div>
+    </div>
+
+     <div class="project" id="project3">
+      <div class="pimage"></div>
+      <div class="p-title">
+        <h2>Tonic</h2>
+      </div>
+      <div class="projectdetails">
+        <ul class="details-list">
+          <li class="client">CANOPY</li>
+          <li class="spacedot"></li>
+          <li class="role">Back End Dev</li>
+          <li class="spacedot"></li>
+          <li class="year">2015</li>
+        </ul>
+      </div>
+      <div class="p-description">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div class="languages">
+        <ul class="languages-list">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <div class="project-access">See project</div>
+    </div>
+
+    <div class="project" id="project4">
+      <div class="pimage"></div>
+      <div class="p-title">
+        <h2>Multi-Post Stories</h2>
+      </div>
+      <div class="projectdetails">
+        <ul class="details-list">
+          <li class="client">CANOPY</li>
+          <li class="spacedot"></li>
+          <li class="role">Back End Dev</li>
+          <li class="spacedot"></li>
+          <li class="year">2015</li>
+        </ul>
+      </div>
+      <div class="p-description">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div class="languages">
+        <ul class="languages-list">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <button class="project-access">See project</button>
+    </div>  
+  </section>
+
+  <section class="about-me">
+    <div class="primary-text-container">
+      <h2 class="primary-text">About<br>Myself</h2>
+    </div>
+    <div class="secondary-text">
+      <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+    <h3 class="connect-text">LET'S CONNECT</h3>
     <div class="socialmedia-icons">
-      <a class="twitter-icon"></a>
-      <a class="linkedin-icon"></a>
-      <a class="m-icon"></a>
-      <a class="cat-icon"></a>
-      <a class="fingers-icon"></a>
+      <ul>
+        <li class="twitter-icon"><a href=""></a></li>
+        <li class="linkedin-icon"><a href=""></a></li>
+        <li class="m-icon"><a href=""></a></li>
+        <li class="cat-icon"><a href=""></a></li>
+        <li class="fingers-icon"><a href=""></a></li>
+      </ul>
```

Comment:
Please follow the design on figma. 
This is your about section:
![Screenshot from 2022-07-07 12-03-57](https://user-images.githubusercontent.com/79506105/177723470-b4187c87-02b6-4011-a11f-8b9562ac0ace.png)

Please stick to this design:



![Screenshot from 2022-07-07 12-06-49](https://user-images.githubusercontent.com/79506105/177723943-19428111-ddbb-419e-bf78-6f0b02d80e7c.png)
 Please carefully pay attention to detail folllow the figma design template you picked.

Reply:
Hello KayLemba:

I appreciate the feedback. After making some changes I suggest you review my project setting the mobile screen width to 375px which is the Figma set width for the mobile version. My project might look different to Figma for wider screen sizes since I made it responsive to different screen sizes according to another feedback that I received before where they encouraged me to set the widths using percentages % to make it more responsive to screen size changes.